### DISCUSSION TITLE
Disable prefetch in Firefox for rel=next header

### DISCUSSION THREAD

**Discussion by espen** [ddnexus/pagy#539] [Q&A]
In development I randomly noticed ?page=2 requests which I did not execute. Looking into this I can see that Firefox will prefetch the rel=next link from the pagy navigation http header. Is there any way to disable this for Firefox? It seems like a bug in Firefox. Anyone else with similar experience?

**Comment by ddnexus** [comment]
Only Firefox?

**Comment by espen** [comment]
From my tests on MacOS with Firefox, Safari and Chrome, the only issue was with Firefox. Just to be clear; this is not an issue with Pagy but since Pagy can use rel=next I thought it was useful to have a discussion here.

**Comment by ddnexus** [comment]
Googling around... it looks like the pre-fetch is considered a feature of Firefox, not a bug.

What is your issue with it?

**Comment by espen** [comment]
Yes I see know that the possibility of[ prefetching next links is the default](http://kb.mozillazine.org/Network.prefetch-next) (this can be disabled but obviously of no use since it is on the client). There is [fetchPriority](https://developer.mozilla.org/en-US/docs/Web/API/HTMLLinkElement/fetchPriority) but it is not supported by Firefox.

My issue was that I was facing some performance issues on an endpoint in my app so was keen on limiting requests. But looks like I was able to fix the performance issues.
